### PR TITLE
Support site maintenance

### LINKED ISSUE #333: h1 and h2 look identical in support pages
This might hurt readability a little bit. It'd be nice if we rendered them slightly differently.

The titles below are respectively h1 and h2.

image

cc @gib

### PR DESCRIPTION
**Pull request by chasballew**
@gib @krallin 

This PR does a few things:

1. Enables table rendering for Redcarpet. We currently use tables, but they don't render. (e.g. https://www.aptible.com/support/topics/account/how-to-prepare-for-training/)
2. Sets a descriptive header for each support topic category in `data/topics.yml` and uses it on the category index pages
3. Updates our article titles and slugs to use Enclave where appropriate
4. Removes the line in `source/stylesheets/base/_typography.scss` that fixed all h1-h6 elements at the same size. This should resolve #333 
5. Increases readability on the categorty index pages by replaces the `h1` with a`<p>` element listing article titles

### CODE REVIEW

File: source/stylesheets/base/_typography.scss [hunk 1]
```diff
@@ -16,7 +16,6 @@ h4,
 h5,
 h6 {
   font-family: $heading-font-family;
-  font-size: modular-scale(1);
```
Comment (gib) [inline source/stylesheets/base/_typography.scss]: I think this is a good change. Will watch for unintended consequences. I think we intenionally specify size on styled headings, so the risk is low.

**Comment by krallin** [comment]
There were some spec errors introduced here: https://travis-ci.org/aptible/www.aptible.com/builds/181010525

I was going to look at these later today, but this is merged now;  @gib were you fixing these in a separate PR?

**Comment by gib** [comment]
Yes, fixing now. Thanks @krallin

**Comment by krallin** [comment]
Perfect; thanks @gib !

**Comment by chasballew** [comment]
Thanks guys!

**Pull request merged**